### PR TITLE
Run pyupgrade to ensure the code is modernized

### PR DESCRIPTION
**Pull request by thedrow**
*Note*: Before submitting this pull request, please review our [contributing
guidelines](https://docs.celeryproject.org/en/master/contributing.html).

## Description

Since we had some external contributions and pyupgrade was improved since the last time we ran it, I made another pass.
The changes in this PR are rather cosmetic but they are still needed.

### CODE REVIEW

**Review by auvipy — APPROVED** [review]

File: celery/utils/text.py [hunk 1]
```diff
@@ -111,7 +111,7 @@ def pretty(value, width=80, nl_width=80, sep='\n', **kw):
     # type: (str, int, int, str, **Any) -> str
     """Format value for printing to console."""
     if isinstance(value, dict):
-        return '{{{0} {1}'.format(sep, pformat(value, 4, nl_width)[1:])
+        return f'{{{sep} {pformat(value, 4, nl_width)[1:]}'
     elif isinstance(value, tuple):
         return '{}{}{}'.format(
```
Comment (maybe-sybr) [inline celery/utils/text.py:116]: May as well update this to an f-string while we're making a diff here

**Review by maybe-sybr — APPROVED** [review]
lgtm too

File: celery/utils/text.py (hunk 1 above)
Comment (thedrow) [reply celery/utils/text.py:116]: Won't it make the line too long?

Comment (maybe-sybr) [reply celery/utils/text.py:116]: oh yeah, seems fair :+1:

**Pull request merged**